src/Organisms/BurgerMenu/BurgerMenu.js

```diff
@@ -28,7 +28,11 @@ const BurgerMenu = ({ open, setOpen }) => {
         <h3>Menu</h3>
       </HeaderMenuStyled>
       <MenuStyled>
-        <Link to="user" className="link" onClick={() => setOpen(false)}>
+        <Link
+          to="/user/:userId"
```

Comment:
"/:userId" must be dynamic:
-url when we came from role page:
![image](https://user-images.githubusercontent.com/92270179/153220972-5a058601-8faa-457a-a9e2-b10af3ac3201.png)
-url when we came from burger menu:
![image](https://user-images.githubusercontent.com/92270179/153221448-f9a47eba-6698-4130-87cd-0cb643476398.png)